medical_pathology/views/medical_pathology_category_view.xml

```diff
@@ -0,0 +1,82 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2016 LasLabs Inc.
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+
+<odoo>
+
+    <record id="medical_pathology_view_category_search" model="ir.ui.view">
+        <field name="name">medical.pathology.category.search</field>
+        <field name="model">medical.pathology.category</field>
+        <field name="arch" type="xml">
+            <search string="Medical Pathology Category">
+                <field name="name" />
+                <field name="parent_id" />
+                <newline />
+                <group expand="0" string="Group By...">
+                    <filter string="Parent Category" domain="[]"
+                        context="{'group_by':'parent_id'}" />
+                </group>
+            </search>
+        </field>
+    </record>
+
+    <record id="medical_pathology_view_category_tree" model="ir.ui.view">
+        <field name="name">medical.pathology.category.tree</field>
+        <field name="model">medical.pathology.category</field>
+        <field name="arch" type="xml">
+            <tree string="Medical Pathology Category">
+                <field name="name" />
+                <field name="parent_id" />
+            </tree>
+        </field>
+    </record>
+
+    <record id="medical_pathology_view_category_form" model="ir.ui.view">
+        <field name="name">medical.pathology.category.form</field>
+        <field name="model">medical.pathology.category</field>
+        <field name="arch" type="xml">
+            <form string="Medical Pathology Category">
+                <header>
```

Comment:
`<header />`

Note to community, blank header is being kept to allow for inherited module injection